spec/factories/training_period_factory.rb

```diff
@@ -8,6 +8,8 @@
     started_on { generate(:base_training_date) }
     finished_on { started_on + 1.day }
 
+    association :expression_of_interest, factory: :lead_provider_active_period
```

Comment:
maybe we can add a trait for linking to a confirmed school partnership?

Reply:
I didn't add this as there isn't a need for it yet but happy to put it in up front 👍